### PR TITLE
Added Windows support to python auth example files

### LINKED ISSUE #10201: [SPIKE] Assist CDC ELIMS team with sending first report: Part 1
### User Story

As a ReportStream engineer, I need to assist the CDC ELIMS team in authenticating and sending RS their first report, if/when they get stuck or have questions.

### Description/Use Case

The CDC ELIMS team is planning to write a Rhapsody script (javascript) to authenticate with RS's server-server auth and send a report to RS via the submission endpoint. Presently, RS has a python example that the team can reference but, if they need help, we should be ready to answer questions and point them in the right direction.

### Acceptance Criteria

-[ ] CDC ELIMS team assisted

### PR DESCRIPTION
**Pull request by thetaurean**
Quality of life improvements for Windows users of the `generate-jwt-example` example project which helps onboard senders to the ReportStream ecosystem. This should get windows users mostly over the fence and also includes some improvements for *nix users (pipx over manually managed virtualenvs)

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Follow the steps in `examples/generate-jwt-python` directory on windows and *nix and make sure it works

## Changes
- Use pipenv for dependency and virtualenv management

### Testing
- [X] Tested locally?
- [x] No Changes ~~Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?~~
- [x] No Changes ~~(For Changes to /frontend-react/...) Ran `npm run lint:write`?~~
- [x] N/A ~~Added tests?~~

### Process
- [x] Are there licensing issues with any new dependencies introduced? No
- [x] Includes a summary of what a code reviewer should test/verify? Yes
- [x] N/A ~~Updated the release notes?~~
- [x] N/A ~~Database changes are submitted as a separate PR?~~
- [x] N/A ~~DevOps team has been notified if PR requires ops support?~~

## Linked Issues
- Fixes #10201 

## Specific Security-related subjects a reviewer should pay specific attention to
No to all questions

### CODE REVIEW

**Review by snesm — APPROVED** [review]

File: prime-router/examples/generate-jwt-python/requirements.txt
Comment (mkalish) [inline prime-router/examples/generate-jwt-python/requirements.txt:1]: Looks like this file isn't used?

File: prime-router/examples/generate-jwt-python/README.md [hunk 1]
```diff
@@ -7,29 +7,70 @@ This directory contains a python example of how to connect to ReportStream as a
...
+      python3 -m pipx ensurepath
+      ```
+    </details>
+
+3. Install the python dependencies using pipx
+    <details>
+      <summary>Windows</summary>
+    
+      ```bash
+      python3.exe -m pipx install PyJWT cryptography
```
Comment (mkalish) [inline prime-router/examples/generate-jwt-python/README.md:39]: This didn't work for me.  I hid two errors:

- it looks like pipx didn't like passing multiple options to install, so at first the command didn't run at all
- I was able to run `python3.exe -m pipx install PyJWT`, but `python3.exe -m pipx install cryptography` did not work, saying there was no associated package

**Review by mkalish — CHANGES_REQUESTED** [review]
Hit a couple of issues, but looking good!  I wasn't able to actually get through running the script on windows though

**Comment by mkalish** [comment]
@thetaurean Just confirming this is ready for a re-review?

File: prime-router/examples/generate-jwt-python/README.md [hunk 2]
```diff
@@ -7,29 +7,70 @@ This directory contains a python example of how to connect to ReportStream as a
...
 
-Then run it:
+Launch a shell and run it:
+
+<details>
+  <summary>Windows</summary>
+
+  ```bash
+  python3.exe -m pipenv shell
+  python3.exe .\generate-jwt.py
```
Comment (mkalish) [inline prime-router/examples/generate-jwt-python/README.md:62]: On my windows machine this didn't quite work... I had to do
```suggestion
  python .\generate-jwt.py
```

**Review by mkalish — APPROVED** [review]
Good to merge, though I think it would make sense to maybe follow up and see if we can drop the `python3` everywhere.  Python2 is no longer supported and most folks should have python pointing at v3

**Pull request merged**